mithril-signer/src/configuration.rs

```diff
@@ -53,11 +55,16 @@ pub struct Configuration {
     #[example = "`mainnet` or `preprod` or `devnet`"]
     pub network: String,
 
-    /// Cardano Network Magic number
+    /// Cardano network magic number
     /// useful for TestNet & DevNet
     #[example = "`1097911063` or `42`"]
     pub network_magic: Option<u64>,
 
+    /// DMQ network magic number
+    /// useful for TestNet & DevNet
```

Comment:
Successive lines in docstrings are concatenated if they are not separated by a blank line, is that what you want ? 
```suggestion
    /// DMQ network magic number
    ///
    /// useful for TestNet & DevNet
```